spec/services/notifications/resolve_path_service_spec.rb

```diff
@@ -32,5 +32,35 @@
         expect(path).to eq(url_helpers.topic_path(topic))
       end
     end
+
+    context "When its a submission comment created notification" do
```

Comment:
We don't tend to capitalize context statements.

Reply:
Yes, This was a mistake.